ci/ciimage/arch/install.sh

```diff
@@ -14,7 +14,7 @@ pkgs=(
   itstool gtk3 java-environment=8 gtk-doc llvm clang sdl2 graphviz
   doxygen vulkan-validation-layers openssh mercurial gtk-sharp-2 qt5-tools
   libwmf cmake netcdf-fortran openmpi nasm gnustep-base gettext
-  python-lxml hotdoc rust-bindgen qt6-base qt6-tools wayland wayland-protocols
+  python-lxml hotdoc qt6-base qt6-tools wayland wayland-protocols
```

Comment:
Why?

Reply:
To clarify why I'm asking this...

Arch ships a new enough bindgen. There's no reason to switch to cargo for this.

The Arch CI fails for totally unrelated reasons, and as a result of that it doesn't help to switch to installing it with cargo because switching to cargo would require updating the image, and if the image could be updated then you wouldn't need cargo.

Fixing the Arch image requires figuring out what's going on with llvm, which is wonderfully wacky and I find llvm very confusing so 🤷. Hopefully someone will figure it out too because static detection is badly misbehaved there, it's not as simple as "it wasn't found".